### DISCUSSION TITLE
Having an issue with installing Git Large File Storage on ubuntu

### DISCUSSION THREAD

**Discussion by Junny1219** [community/community#23497] [New to GitHub]
<aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/69666617/181077421-d1a1b5eb-8876-4e67-9ce7-69bd491d7687.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage" target="_blank" rel="noopener nofollow ugc">Installing Git Large File Storage - GitHub Docs</a></h3>

  <p>In order to use Git LFS, you'll need to download and install a new program that's separate from Git.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
<p>
From this website, I followed the instruction until (5) and I am getting an error on cmd<br>
./install.sh: line 22: git: command not found</p>
<p>git-lfs-linux-amd64-v3.1.4 file is located in the Download file and the current working directory is set to<br>
user@user-VirtualBox:~/Downloads/git-lfs-linux-amd64-v3.1.4$<br>
and typed $ sudo ./install.sh<br>
It would give me an error (./install.sh: line 22: git: command not found)</p>
<p>any help would be appreciated.<br>
I need to upload a large file 200~300mb file for the project jsonl, CSV, so i really need to fix this problem asap.</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="Junny1219" data-post="1" data-topic="251779">
<div class="title">
<div class="quote-controls"></div>
 Junny1219:</div>
<blockquote>
<p>./install.sh: line 22: git: command not found</p>
</blockquote>
</aside>
<p>This sounds a lot like you haven’t installed Git on the system yet. Install the <code>git</code> package.</p>
<p>By the way there’s also a <code>git-lfs</code> package, which is preferable to manual installation.</p>
